dist/index.js

```diff
@@ -1767,7 +1767,7 @@ function onceStrict (fn) {
 // We use any as a valid input type
 /* eslint-disable @typescript-eslint/no-explicit-any */
 Object.defineProperty(exports, "__esModule", { value: true });
-exports.toCommandValue = void 0;
+exports.toCommandProperties = exports.toCommandValue = void 0;
```

Comment:
Do we need to check this in? The build dist PR should create this after we update lockfile.

Reply:
Oops, will revert